tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -283,13 +283,15 @@ public async Task ExecuteAScheduledJobWithDateTimeOffsetInThePast()
     [Fact]
     public async Task WhileAwaitingJobTriggeringInstantJobShouldAnywayTriggerCronJob()
     {
-        ServiceCollection.AddNCronJob(n => n.AddJob<SimpleJob>(p => p.WithCronExpression("0 * * * *")));
+        ServiceCollection.AddNCronJob(n => n.AddJob<SimpleJob>(p => p.WithCronExpression(Cron.AtMinute0)));
         var provider = CreateServiceProvider();
 
         (IDisposable subscription, IList<ExecutionProgress> events) = RegisterAnExecutionProgressSubscriber(provider);
 
         await provider.GetRequiredService<IHostedService>().StartAsync(CancellationToken);
 
+        FakeTimer.Advance(TimeSpan.FromSeconds(1));
```

Comment:
Now required to ensure the following assertions
- `Scheduled orchestration should have started before the instant job related one...`
- `...and cancelled before the initialization of the instant job related one.`